java/java.sourceui/src/org/netbeans/modules/java/source/ui/LspElementUtils.java

```diff
@@ -294,40 +294,42 @@ private static String createDetail(CompilationInfo ci, Element original) {
     }
     
     private static StructureProvider.Builder processOffsetInfo(Object[] info, StructureProvider.Builder builder) {
-        int selStart = (int)info[3];
-        if (selStart < 0) {
-            selStart = (int)info[1];
-        }
-        int selEnd = (int)info[4];
-        if (selEnd < 0) {
-            selEnd = (int)info[2];
-        }
-        TreePathHandle pathHandle = (TreePathHandle)info[6];
-        FileObject f = (FileObject)info[0];
-        boolean[] synthetic = new boolean[] { false };
-        if (f != null) {
-            builder.file(f);
-            if (pathHandle != null) {
-                try {
-                    JavaSource js = JavaSource.forFileObject(f);
-                    if (js == null) {
-                        return null;
+        if (info != null) {
```

Comment:
Note: I have fixed the same thing in #3979 , possible clash here.

Reply:
Conflict resolved.